mobile/src/reducers/auth/auth.epics.ts

```diff
@@ -0,0 +1,26 @@
+import { ActionsObservable, ofType, combineEpics } from 'redux-observable';
+import { Observable } from 'rxjs/Observable';
+import { DependenciesContainer, AppState } from '../app.reducer';
+import { PressLogIn, PressLogOut, pressLogIn, pressLogOut } from '../auth/auth.action';
+
+export const pressLogInEpic$ = (action$: ActionsObservable<PressLogIn>, state: AppState, dep: DependenciesContainer) =>
+    action$.ofType('PRESS-LOG-IN')
+        .do(x => {
+            dep.oauthProcess.login();
+        }).ignoreElements();
+
+export const pressLogOutEpic$ = (action$: ActionsObservable<PressLogOut>, state: AppState, dep: DependenciesContainer) =>
+    action$.ofType('PRESS-LOG-OUT')
+        .do(x => {
+            dep.oauthProcess.logout();
+
```

Comment:
clean empty lines up. Also, you can probably get rid of {} as those are one-liners